Automattic/wp-calypso
# Posts: remove 'Latest Drafts' column.

This PR removes the 'Latest Drafts' column from the `/posts/` screen. The feature is duplicated in the MasterBar directly above.

**Before:**
<img width="1596" alt="screencapture_at_mon_oct_16_14_38_46_edt_2017" src="https://user-images.githubusercontent.com/942359/31955934-f066ee6a-b8b7-11e7-9437-36d54f4682c2.png">

**After:**
<img width="1596" alt="screencapture_at_mon_oct_16_14_38_28_edt_2017" src="https://user-images.githubusercontent.com/942359/31955943-f60a0564-b8b7-11e7-82a4-c06169665a99.png">

**To test:**
1. Select a site.
2. Visit your Blog Posts list.
3. Verify that the 'Latest Drafts' sidebar doesn't render at any browser size.
